sdk/tests/tutorials/ibor/test_bitemporal.py

```diff
@@ -35,14 +35,17 @@ def tearDownClass(cls):
         delete_entities(cls.id_generator)
 
     def print_transactions(self, as_at, transactions):
-        print("transactions at: {}".format(as_at))
-
-        for transaction in transactions:
-            print("{}\t{}\t{}\t{}\t{}".format(transaction.instrument_uid,
-                                              transaction.transaction_date,
-                                              transaction.units,
-                                              transaction.transaction_price.price,
-                                              transaction.total_consideration.amount))
+        # print("transactions at: {}".format(as_at))
```

Comment:
delete the commented code/function

Reply:
ah, missed it.  Will take care of.

Reply:
oh, no, I left that in there in case someone wanted it back.  I'm not a fan of print statements in tests or code as you know.  :|
Would it be best to revert the commenting or leave as is, or remove the stuff that I did comment, which may propagate to removing the method, and subsequently changing a number of tests???

Reply:
Gone